airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionManagerWorkflowImpl.java

```diff
@@ -398,4 +333,140 @@ private void continueAsNew(final ConnectionUpdaterInput connectionUpdaterInput)
     }, input);
   }
 
+  /**
+   * Return the duration to wait. This is calculated by the configFetchActivity and return the
+   * duration to wait until the next run
+   *
+   * This return the time to wait since the last successful run based on the schedule. If the workflow
+   * is manual or disabled the time to wait will be infinite.
```

Comment:
```suggestion
   * Calculate the duration to wait so the workflow adheres to its schedule. This lets us 'schedule' the next run.
   * 
   * This is calculated by the configFetchActivity and depends on the last successful run and the schedule.
   * 
   *  Wait time is infinite If the workflow is manual or disabled since we never want to schedule this.
```

Reply:
Done